securebanking-openbanking-uk-rcs-api/src/main/java/com/forgerock/securebanking/openbanking/uk/rcs/api/dto/consent/details/AccountsConsentDetails.java

```diff
@@ -36,30 +33,16 @@
 @SuperBuilder
 @AllArgsConstructor
 @NoArgsConstructor
+@Component
```

Comment:
Shouldn't be a component, this will not be a spring managed bean (same goes for all of the ConsentDetails implementations).

Reply:
Yes, I agree.